src/server/req_manager.c

```diff
@@ -3559,16 +3552,17 @@ int
 is_entity_resource_set(attribute *pattr, char *resc_name)
 {
 	if (pattr->at_flags & ATR_VFLAG_SET) {
-		pbs_entlim_key_t  *pkey = NULL;
+		char *key = NULL;
 		void *ctx = pattr->at_val.at_enty.ae_tree;
 		char resc[PBS_MAX_RESC_NAME+1];
 
-		while ((pkey=entlim_get_next(pkey, ctx)) != NULL) {
-			if (entlim_resc_from_key(pkey, resc, PBS_MAX_RESC_NAME) == 0)
-				if (strcmp(resc, resc_name) == 0) {
-					free(pkey);
-					return 1;
+		if (entlim_get_first(ctx, (void **)&key) != NULL) {
```

Comment:
why have to changed the entlim functions? Looks like earlier code with just entlim_get_next() was good enough!

Reply:
This was changed since the earlier version of pbs_idx_find was not returning key back, but since we changed that, I think I can revert this change also...